code/modules/power/TEG.dm

```diff
@@ -99,6 +103,17 @@
 			src.serial_num += "-[variant_b]"
 			variant_b_active = TRUE
 
+	proc/toggle_active()
+		src.target_pressure_enabled = !src.target_pressure_enabled
+		logTheThing(LOG_STATION, src, "toggled blower power [src.target_pressure_enabled ? "on" : "off"] using mechcomp.")
```

Comment:
Maybe the proc name should be more specific if it's exclusively for mechcomp? Like `mechcomp_toggle_active()` or something

Just future-proofing so no one accidentally uses it for something else (which is extraordinarily unlikely but still)